tenma/tenmaDcLib.py

```diff
@@ -362,9 +444,10 @@ def setOCP(self, enable=True):
 
             :param enable: Boolean to enable or disable
         """
-        conf = 1 if enable else 0
-        command = "OCP{conf}".format(conf=conf)
-        self.__sendCommand(command)
+        enableFlag = 1 if enable else 0
```

Comment:
similar, I'm not too fond of change for the sake of change :-)

Reply:
Oh, this was more because the function was using `conf` for the variable but it's not a save or recall operation. I can change the code back, but I think renaming the variable makes more sense

Reply:
In this case you're clearly right that `enableFlag` sounds better than `conf`